### DISCUSSION TITLE
Error: Get "http://unix.socket/1.0": if launch lxc list

### DISCUSSION THREAD

**Discussion by diashtrih** [ganto/copr-lxc4#28] [Q&A] [answered]
```
Error: Get "http://unix.socket/1.0": dial unix /var/lib/lxd/unix.socket: connect: no such file or directory
```
error if run "sudo lxс list"not from root account

**Comment by diashtrih** [comment]
sudo lxd init, is same

**Comment by ganto** [comment, marked as answer]
The LXD_SOCKET environment variable must be set to `/run/lxd.socket`. By default this is achieved by sourcing `/etc/profile.d/lxd.sh`. If this file is missing or your shell or application is not loading it, you must manually set the variable.

```
$ lxc list
Error: Get "http://unix.socket/1.0": dial unix /var/lib/lxd/unix.socket: connect: no such file or directory
$ cat /etc/profile.d/lxd.sh 
# /etc/profile.d/lxd.sh - Set LXD socket path

LXD_SOCKET=/run/lxd.socket
export LXD_SOCKET
$ . /etc/profile.d/lxd.sh 
$ lxc list
+------+-------+------+------+------+-----------+
| NAME | STATE | IPV4 | IPV6 | TYPE | SNAPSHOTS |
+------+-------+------+------+------+-----------+
```

**Comment by ganto** [comment]
Btw, this is also described in the [wiki: Getting started with LXD on Fedora](https://github.com/ganto/copr-lxc4/wiki/Getting-Started-with-LXD-on-Fedora) page.
